.github/workflows/test.yml

```diff
@@ -46,7 +46,7 @@ jobs:
           check-latest: true
           cache: true
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@v3.7.0
+        uses: golangci/golangci-lint-action@v6.1.0
         with:
           version: v1.55.2
```

Comment:
This has to be updated
https://github.com/golangci/golangci-lint/releases/tag/v1.60.3

Reply:
updated, thanks